dids/src/main/kotlin/web5/sdk/dids/methods/jwk/DidJwk.kt

```diff
@@ -144,36 +144,33 @@ public class DidJwk(uri: String, keyManager: KeyManager) : Did(uri, keyManager)
         throw IllegalArgumentException("decoded jwk value cannot be a private key")
       }
 
-      val verificationMethodId = URI.create("$did#0")
-      val verificationMethod = VerificationMethod.builder()
+      val verificationMethodId = "$did#0"
+      val verificationMethod = VerificationMethod.Builder()
         .id(verificationMethodId)
-        .publicKeyJwk(publicKeyJwk.toJSONObject())
-        .controller(URI(did))
+        .publicKeyJwk(publicKeyJwk)
+        .controller(did)
         .type("JsonWebKey2020")
         .build()
 
-      val verificationMethodRef = VerificationMethod.builder()
-        .id(verificationMethodId)
-        .build()
-
-      val didDocumentBuilder = DIDDocument.builder()
-        .contexts(
-          mutableListOf(
-            URI.create("https://w3id.org/security/suites/jws-2020/v1")
-          )
-        )
-        .id(URI(did))
-        .verificationMethod(verificationMethod)
+      val didDocumentBuilder = DIDDocument.Builder()
+        .context("https://www.w3.org/ns/did/v1")
```

Comment:
this needs to be removed - no context property for this method

Reply:
hmm, that is not what the did core spec says - https://www.w3.org/TR/did-core/#production-0 

> The serialized value of @context MUST be the [JSON String](https://www.rfc-editor.org/rfc/rfc8259#section-7) https://www.w3.org/ns/did/v1, or a [JSON Array](https://www.rfc-editor.org/rfc/rfc8259#section-5) where the first item is the [JSON String](https://www.rfc-editor.org/rfc/rfc8259#section-7) https://www.w3.org/ns/did/v1 and the subsequent items are serialized according to the JSON [representation](https://www.w3.org/TR/did-core/#dfn-representations) [production](https://www.w3.org/TR/did-core/#dfn-production) rules.

is there a did jwk spec language i'm missing here?

Reply:
> JSON-LD production MUST include the...

This is not a JSON-LD method. This is a JSON method -- https://www.w3.org/TR/did-core/#json